content/apps/_index.md

```diff
@@ -0,0 +1,157 @@
+
+* the ability to interactively ask questions to generate `values.yaml` based on JSON Schema
+* the ability to create pull requests against the GitOps repo that manages your team/cluster
+* the ability to store secrets in vault
+* the ability to upgrade all apps to the latest version
+
+Planned features include:
+
+* integrating [kustomize](https://github.com/kubernetes-sigs/kustomize) to allow existing charts to be modified
+* storing Helm repository credentials in vault
+* taking existing `values.yaml` as defaults when asking questions based on JSON Schema during app upgrade
+* only asking new questions during app upgrade
+* `jx get apps` - the ability to list all apps that can be installed
+* integration for bash completion
+
+# APIs
+
+APIs can be consumed from a microservice deployed to Kubernetes, from outside the cluster (with appropriate 
+configuration for authentication) or from inside pipeline extensions.
+
+Jenkins X APIs uses [Kubernetes Custom Resources](https://kubernetes
+.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to expose its API. The API is declarative, and 
+Jenkins X uses controllers to try to match the actual state to the desired state. Jenkins X continually takes action 
+to achieve and maintain the desired state.
+
+Jenkins X uses this architectural pattern itself meaning that this API is very complete.
+
+Custom resources are exposed by the Kubernetes API server and can be accessed as REST endpoints, using `kubectl` or 
+via client libraries for different platforms. Currently we offer clients for:
+
+* Go
+* Java (incomplete) - [Quickstart](https://github.com/jenkins-x-quickstarts/spring-boot-watch-pipeline-activity)
+* Javascript [Quickstart](https://github.com/jenkins-x-quickstarts/spring-boot-watch-pipeline-activity)
+
+## Source code management
+
+The ability to add new SCM connections to Jenkins X from an app is under active development.
+
+## Identity Management
+
+In the future we intend to add the ability to plug your own identity management and SSO solution in to Jenkins X.
+
+## Custom Quickstarts
+
+Jenkins X ships with a series of quickstarts that provide you with pre-made applications that you can start a project with.
+
+You can create your own quickstarts that give your team members a starting point. Read more about [creating quickstarts](/developing/create-quickstart).
+
+You can customize the list of quickstarts available to your team making it easier for them to select the right place to start. Read more about [customizing the quickstarts available](/developing/create-quickstart#customising-your-teams-quickstarts).
+
+
+
+# Pipeline Extensibility
+
+## Pipeline triggers
+
+TODO
+
+## Build environment (creds, env vars, parameters)
+
+### Build Packs & Pod Templates
+
+In Jenkins X a _Build Pack_ allows you to transform source code into a applications which can be deployed on Kubernetes. Build Packs are based on [draft](https://draft.sh/), and will automatically add:
```

Comment:
"In Jenkins X a _Build Pack_ allows you to transform source code into a applications..." reads odd.  "In Jenkins X a _Build Pack_ allows you to transform source code into applications..."